whatwg.org/ipr-policy

```diff
@@ -113,11 +114,11 @@
 <p>When a <a href="workstream-policy#workstream-participant">Workstream Participant</a> joins a <a href="workstream-policy#workstream">Workstream</a>, the most-recent <a href="workstream-policy#review-draft">Review Draft</a> is deemed to have been "published" and a <a href="ipr-policy#25-patent-exclusion-period">Patent Exclusion Period</a> commences, with respect to that <a href="workstream-policy#workstream-participant">Workstream Participant</a>, when it joins.</p>
 <h4 id="55-patent-exclusions">5.5. Patent exclusions<a class="self-link" href="#55-patent-exclusions"></a></h4>
 <h5 id="551">5.5.1.<a class="self-link" href="#551"></a></h5>
-<p>A <a href="workstream-policy#workstream-participant">Workstream Participant</a> may exclude <a href="#22-essential-patent-claims">Essential Patent Claims</a> from the <a href="#53-review-draft-licensing-obligations">Review Draft Licensing Obligations</a> by providing a <a href="ipr-policy#27-patent-exclusion-notice">Patent Exclusion Notice</a> to the <a href="workstream-policy#editor">Workstream</a> and the <a href="sg-agreement#steering-group">Steering Group</a> within the <a href="ipr-policy#25-patent-exclusion-period">Patent Exclusion Period</a>. (A <a href="workstream-policy#workstream-participant">Workstream Participant</a> may file a <a href="ipr-policy#27-patent-exclusion-notice">Patent Exclusion Notice</a> and continue to participate in the <a href="workstream-policy#workstream">Workstream</a>.)</p>
+<p>A <a href="workstream-policy#workstream-participant">Workstream Participant</a> may exclude <a href="#22-essential-patent-claims">Essential Patent Claims</a> from the <a href="#53-review-draft-licensing-obligations">Review Draft Licensing Obligations</a> by providing a <a href="ipr-policy#27-patent-exclusion-notice">Patent Exclusion Notice</a> to the <a href="workstream-policy#workstream">Workstream</a> <a href="workstream-policy#editor">Editor</a> and the <a href="sg-agreement#steering-group">Steering Group</a> within the <a href="ipr-policy#25-patent-exclusion-period">Patent Exclusion Period</a>. (A <a href="workstream-policy#workstream-participant">Workstream Participant</a> may file a <a href="ipr-policy#27-patent-exclusion-notice">Patent Exclusion Notice</a> and continue to participate in the <a href="workstream-policy#workstream">Workstream</a>.)</p>
```

Comment:
This seems to fix an actual issue in that previously it said "Workstream" and now it says "Workstream Editor".